resources/energyplus/ProposedEnergy+.idd

```diff
@@ -13114,10 +13119,17 @@ ZoneInfiltration:DesignFlowRate,
        \note  "C" in Equation
        \type real
        \default 0
-  N8 ; \field Velocity Squared Term Coefficient
+  N8 , \field Velocity Squared Term Coefficient
        \note  "D" in Equation
        \type real
        \default 0
+  A5 ; \field Density Basis
+       \note The air density to use when converting from volume flow to mass flow.
+       \type choice
+       \key Outdoor
+       \key Standard
+       \key Indoor
+       \default Outdoor
```

Comment:
This is E+ `ZoneInfiltration:DesignFlowRate`, it hasn't been added in OS SDK, and it should be added to OS:SpaceInfiltration:DesignFlowRate

Reply:
See ea567ca36148a50803e6a531999c62529dde5715